test/test_util.py

```diff
@@ -64,3 +64,14 @@ def test_timing_can_use_int(self):
             _create_bus_config({**self.base_config, **timing_conf})
         except TypeError as e:
             self.fail(e)
+
+
+class TestChannel2Int(unittest.TestCase):
+    def test_channel2int(self) -> None:
+        self.assertEqual(0, channel2int("can0"))
+        self.assertEqual(0, channel2int("vcan0"))
+        self.assertEqual(1, channel2int("vcan1"))
+        self.assertEqual(12, channel2int("vcan12"))
+        self.assertEqual(3, channel2int(3))
```

Comment:
I'd add this case too:

```suggestion
        self.assertEqual(3, channel2int(3))
        self.assertEqual(42, channel2int("42"))
```